openc3-ruby/Dockerfile

```diff
@@ -82,6 +84,9 @@ ENV IMAGE_USER=openc3
 ENV IMAGE_GROUP=openc3
 ENV USER_ID=1001
 ENV GROUP_ID=1001
+# The build time venv is at /openc3/venv and the runtime venv is at /gems/python_packages
+# NOTE: This is delared as PYTHONUSERBASE: "/gems/python_packages" in compose.yaml
+ENV PYTHONPATH=/openc3/venv/lib/python3.11/site-packages/:/gems/python_packages/lib/python3.11/site-packages/
```

Comment:
I was able to keep the existing `PYTHONUSERBASE` definition of the python packages doing this